### PR TITLE
usb: device: Use sys_le16_to_cpu conversion

### PR DESCRIPTION
**Pull request by finikorg**
Use sys_le16_to_cpu conversion

### CODE REVIEW

**Comment by mped-oticon** [comment]
please add a PR description, otherwise looks fine to me.

File: subsys/usb/usb_device.c [hunk 1]
```diff
@@ -179,9 +180,9 @@ static void usb_print_setup(struct usb_setup_packet *setup)
 	LOG_DBG("Setup: %x %x %x %x %x",
 		setup->bmRequestType,
 		setup->bRequest,
```
Comment (mped-oticon) [inline subsys/usb/usb_device.c]: `bRequest` and `bmRequestType` are already in host-endianess?

**Review by mped-oticon — COMMENTED** [review]
lgtm

File: subsys/usb/usb_device.c (hunk 1 above)
Comment (finikorg) [reply subsys/usb/usb_device.c]:
```
/** setup packet definitions */
struct usb_setup_packet {
	u8_t bmRequestType;  /**< characteristics of the specific request */
	u8_t bRequest;       /**< specific request */
	u16_t wValue;        /**< request specific parameter */
	u16_t wIndex;        /**< request specific parameter */
	u16_t wLength;       /**< length of data transferred in data phase */
};
```

**Review by mped-oticon — APPROVED** [review]

**Review by pawelzadrozniak — APPROVED** [review]

**Pull request merged**